src/Microsoft.Health.Fhir.CosmosDb/Features/Search/Queries/QueryBuilder.cs

```diff
@@ -148,9 +145,6 @@ private void AppendSystemDataFilter(string keyword = null)
 
                 _queryBuilder
                     .Append("(")
```

Comment:
We don't need this extra parenthese anymore since there is only a single condition. Also, i would remove the keyword parameter above and just directly put "WHERE" instead here since it's no longer needed to have different keyword.